tools/astarte_export/lib/export.ex

```diff
@@ -0,0 +1,348 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2019 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Export do
+  alias Astarte.Export.FetchData
+  alias Astarte.Export.XMLGenerate
+  require Logger
+
+  @moduledoc """
+    This  module provide API functions to export realm device 
+    data in a xml format. This data can be used by astarte_import 
+    application utlity  to import into a new realm.
+  """
+
+  @doc """
+  The export_realm_data/2 function required 2 arguments to export 
+  the realm data into XML format.
+  the arguments are
+   -realm-name -> This is a string format of input
```

Comment:
Please, try to be consistent with spacing.

Reply:
Did the necessary changes